drupal/.gitignore

```diff
@@ -0,0 +1,28 @@
+# Ignore directories generated by Composer
+/.editorconfig
+/.gitattributes
+/vendor/
+/web/_ping.php
+/web/.gitignore
+/web/core/
+/web/modules/.gitignore
+/web/modules/contrib/
+/web/themes/.gitignore
+/web/themes/contrib/
+/web/profiles/.gitignore
+/web/profiles/contrib/
+/web/libraries/
+/web/sites/.gitignore
+/web/sites/default/.gitignore
```

Comment:
Could use /web/**/.gitignore to shave off a few lines instead of listing all the particular directories separately

Reply:
done in https://github.com/wunderio/next4drupal-project/pull/10/commits/a7db4595229392194f98077a01456768b90f09e8